[2024-01-30T17:38:04.761+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-28T23:00:00+00:00 [queued]>
[2024-01-30T17:38:04.884+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-28T23:00:00+00:00 [queued]>
[2024-01-30T17:38:04.887+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-30T17:38:05.042+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): get_weather_data> on 2024-01-28 23:00:00+00:00
[2024-01-30T17:38:05.064+0000] {standard_task_runner.py:60} INFO - Started process 2795 to run task
[2024-01-30T17:38:05.075+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_api_dag', 'get_weather_data', 'scheduled__2024-01-28T23:00:00+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/weather_api_dag.py', '--cfg-path', '/tmp/tmp4mnzyfkq']
[2024-01-30T17:38:05.088+0000] {standard_task_runner.py:88} INFO - Job 54: Subtask get_weather_data
[2024-01-30T17:38:05.248+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-28T23:00:00+00:00 [running]> on host 68c8327b56d3
[2024-01-30T17:38:06.041+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_dag' AIRFLOW_CTX_TASK_ID='get_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-28T23:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-28T23:00:00+00:00'
[2024-01-30T17:38:06.831+0000] {logging_mixin.py:188} INFO - {'cod': 401, 'message': 'Invalid API key. Please see https://openweathermap.org/faq#error401 for more info.'}
[2024-01-30T17:38:10.185+0000] {logging_mixin.py:188} INFO - {'record': {'cod': 401, 'message': 'Invalid API key. Please see https://openweathermap.org/faq#error401 for more info.'}, 'metadata': {'id': '65b93402266cfc3fde8362e4', 'createdAt': '2024-01-30T17:38:10.191Z', 'private': True, 'name': 'weather_38d016c4-3d2f-4979-bb55-5453a87d3e1a', 'collectionId': '65ab4466266cfc3fde7c6f31'}}
[2024-01-30T17:38:10.197+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-30T17:38:10.280+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_api_dag, task_id=get_weather_data, execution_date=20240128T230000, start_date=20240130T173804, end_date=20240130T173810
[2024-01-30T17:38:10.407+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-30T17:38:10.471+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
